format/index.md

```diff
@@ -785,6 +794,15 @@ include additional [fields] in the response.
 GET /articles?include=author&fields[articles]=title,body&fields[people]=name
 ```
 
+If both query parameters `include` and `fields[TYPE]` are provided by the client, 
+values of `TYPE` **MUST** either refer to the primary data or be listed in 
+`include`, otherwise the server **MUST** return `400 Bad Request`.
```

Comment:
I'd like this paragraph removed. I think this is an unnecessary restriction. Consider heterogeneous collections: the client may not know the members' types in the response.